windows-server-container-tools/Install-ContainerHost/Install-ContainerHost.ps1

```diff
@@ -77,7 +77,7 @@ param(
 
     [string]
     [ValidateNotNullOrEmpty()]
-    $WimPath = "http://aka.ms/ContainerOSImage"
+    $WimPath = "https://aka.ms/ContainerOSImage"
```

Comment:
This URL is redirected back to an http URL http://download.microsoft.com/download/...